.golangci.yml

```diff
@@ -32,6 +32,26 @@ issues:
     - linters:
         - govet
       text: "composites"
+    - linters:
+        - staticcheck
+      path: adapters/handlers/grpc/v1/prepare_reply.go
+      text: 'SA1019' # TODO: remove this once deprecated gRPC fields are removed from API
+    - linters:
+        - staticcheck
+      path: adapters/handlers/grpc/v1/parse_search_request.go
+      text: 'SA1019' # TODO: remove this once deprecated gRPC fields are removed from API
+    - linters:
+        - staticcheck
+      path: adapters/handlers/grpc/v1/batch_parse_request.go
+      text: 'SA1019' # TODO: remove this once deprecated gRPC fields are removed from API
+    - linters:
+        - staticcheck
+      path: adapters/handlers/grpc/v1/service.go
+      text: 'SA1019' # TODO: remove this once deprecated gRPC fields are removed from API
+    - linters:
+        - staticcheck
+      path: test/acceptance/grpc/grpc_test_deprecated.go
```

Comment:
The problem is that I want to mark the gRPC fields in the `.proto` as deprecated so that consuming clients know this. However, since our code depends on the auto-generated stubs (which also show the deprecation) the linter flags this as an issue that needs fixing. So I added this targeted excludes to pass the linter